shared/transforms/shared_shorthand2xccdf.xslt

```diff
@@ -390,14 +390,7 @@
           <xsl:attribute name="system">ocil-transitional</xsl:attribute>
           <check-export>
 
-            <xsl:attribute name="export-name">
-              <!-- add clauses if specific macros are found within -->
-              <xsl:if test="fileperms-check-macro or fileowner-check-macro or filegroupowner-check-macro">it does not</xsl:if>
-              <xsl:if test="service-disable-check-macro">the service is running</xsl:if>
-              <xsl:if test="socket-disable-check-macro">the socket is running</xsl:if>
-              <xsl:if test="service-enable-check-macro">the service is not running</xsl:if>
-              <xsl:if test="module-disable-check-macro">no line is returned</xsl:if>
-            </xsl:attribute>
+            <xsl:attribute name="export-name"/>
```

Comment:
Is this attribute without value needed?
I think this can be removed.

Reply:
Don't ask me why, but it looks like it is :D When I removed it, it failed to build...

Reply:
NACK, this needs to be fixed for real

Reply:
Sorry, I don't understand what does it mean "fixed for real". This PR is meant to remove macro artefacts no longer in use. It's not fixing anything.

Reply:
AFAICT the macros were replaced correctly using Jinja, so there is nothing to fix.

Reply:
I mean removing the attribute completely, don't leave it there because it doesn't build. We need to understand why it fails and remove all traces of it.

Reply:
Ah, ok. Makes sense

Reply:
OCIL questions end with sentence
`Is it the case that` which continues with the contents of "export-name". So the attribute is still needed, unless we'll remove it altogether and update OCIL question directly. So I cannot remove the attribute in this "cleanup" PR. I suggest to investigate streamlining OCIL generation is separate effort.

Reply:
well OK but then you broke that with this PR so we can't merge it

Reply:
I am not aware of any breakage - part I remove in the PR is replacement of macros with texts. With macros rewrite to jinja, there are no more macros to replace in this phase.
`xccdf-unlinked-resolved.xml` does not differ between master and my PR.

Reply:
@mpreisler thanks for tip how to make the code slightly nicer. Updated.